src/v/redpanda/BUILD

```diff
@@ -95,11 +96,33 @@ redpanda_cc_library(
     ],
 )
 
+bool_flag(
+    name = "lto",
+    build_setting_default = True,
+)
+
+config_setting(
+    name = "use_lto",
+    flag_values = {
+        ":lto": "true",
+    },
+    visibility = ["//visibility:public"],
+)
+
 redpanda_cc_binary(
     name = "redpanda",
     srcs = [
         "main.cc",
     ],
+    linkopts = [] +
```

Comment:
Nit: you don't need to add empty array, I am assuming this was left from debugging/dev